### PR TITLE
prep LunrCoreLmdb for NuGet

### PR DESCRIPTION
**Pull request by danielcrenna**
This is a PR for https://github.com/bleroy/lunr-core/issues/28.

It prepares `LunrCoreLmdb` to be a NuGet package.

If you can push it out the same way you're pushing out `LunrCore`, that would be great.

For now I can use the offline package for testing & development.

### CODE REVIEW

File: LunrCoreLmdb/LunrCoreLmdb.csproj [hunk 1]
```diff
@@ -1,18 +1,36 @@
...
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <LangVersion>8</LangVersion>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <RootNamespace>Lunr</RootNamespace>
+    <Version>2.3.8.5</Version>
+    <Authors>Bertrand Le Roy</Authors>
+    <Company>Decent Consulting</Company>
+    <Description>A .NET Core, LMDB-backed port of Oliver Nightingale's lunr.js library, a lightweight full-text indexing library that is "a bit like Solr, but much smaller and not as bright." Icon adapted from https://commons.wikimedia.org/wiki/File:Internal_Structure_of_the_Moon.JPG by Iqbal Mahmud under Creative Commons Attribution Share Alike 4.0 International</Description>
```
Comment (bleroy) [inline LunrCoreLmdb/LunrCoreLmdb.csproj:9]: Let's put you as the main author for this package and remove Decent Consulting. Let's change the description to something like "LMDB storage add-on for LunrCore [...]". Also add yourself to the copyright.

**Review by bleroy — CHANGES_REQUESTED** [review]

**Comment by danielcrenna** [comment]
I will make these changes but I'm going to hold off until my mutable index feature is completed and tested, which should be done this week.

**Comment by NimaAra** [comment]
@danielcrenna Any update on the mutable index feature please?

File: LunrCoreLmdb/LunrCoreLmdb.csproj [hunk 2]
```diff
@@ -1,18 +1,36 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <LangVersion>8</LangVersion>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <RootNamespace>Lunr</RootNamespace>
+    <Version>2.3.8.5</Version>
+    <Authors>Bertrand Le Roy</Authors>
```
Comment (bleroy) [inline LunrCoreLmdb/LunrCoreLmdb.csproj:7]:
```suggestion
    <Authors>Daniel Crenna, Bertrand Le Roy</Authors>
```

**Comment by bleroy** [comment]
I'll merge that for now, for the sake of having a package on NuGet, which is long overdue. Let's take the mutable index changes when they're done, separately.

**Review by bleroy — APPROVED** [review]

**Pull request merged**